src/Phinx/Db/Adapter/AdapterInterface.php

```diff
@@ -153,15 +153,6 @@ public function getOutput();
      */
     public function migrated(MigrationInterface $migration, $direction, $startTime, $endTime);
 
-    /**
-     * Toggle a migration breakpoint.
-     *
-     * @param \Phinx\Migration\MigrationInterface $migration
-     *
-     * @return \Phinx\Db\Adapter\AdapterInterface
-     */
-    public function toggleBreakpoint(MigrationInterface $migration);
```

Comment:
We are removing interface methods here on master?

Reply:
reversed, going to open a PR on 0.next for this later.